src/panoptes/utils/horizon.py

```diff
@@ -40,51 +106,45 @@ def __init__(self, obstructions=list(), default_horizon=30):
         """
         super().__init__()
 
-        obstruction_list = list()
-        for obstruction in obstructions:
-            assert isinstance(obstruction, list), "Obstructions must be lists"
-            assert len(obstruction) >= 2, "Obstructions must have at least 2 points"
-
-            obstruction_line = list()
-            for point in obstruction:
-                assert isinstance(point, list), "Obstruction points must be lists"
-                assert len(point) == 2, "Obstruction points must be 2 points"
+        # Parse obstruction list
+        if obstructions is None:
+            obstructions = []
+        self.obstructions = [Obstruction(_) for _ in obstructions]
 
-                assert type(point[0]) is not bool, "Bool not allowed"
-                assert type(point[1]) is not bool, "Bool not allowed"
-                assert isinstance(point[0], (float, int, np.integer)), "Must be number-like"
-                assert isinstance(point[1], (float, int, np.integer)), "Must be number-like"
+        # Add default horizon
+        self._default_horizon = get_quantity_value(default_horizon, "deg") * u.deg
 
-                alt = float(point[0])
-                az = float(point[1])
+        # Calculate horizon at each integer azimuth
+        # This is included for backwards compatibility with POCS
+        self.horizon_line = np.zeros(360, dtype="float")
+        for i, az in enumerate(self.horizon_line):
+            self.horizon_line[i] = self.get_horizon(az).to_value(u.deg)
```

Comment:
The `horizon_line` here switched between being an array of altitudes instead of an array of azimuths.

```suggestion
        # Calculate horizon at each integer azimuth
        self.alt_line = np.ones(360) * self._default_horizon
        self.az_line = np.linspace(0, 360, 360)
        
        for i, az in enumerate(self.az_line):
            self.alt_line[i] = self.get_horizon(az).to_value(u.deg)
            
        # Assign to legacy name for backwards-compatibility.
        self.horizontal_line = self.alt_line
```